expanalysis/results.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 24 16:03:04 2016
+
+@author: ian
```

Comment:
You should replace these headers (from your IDE?) with something akin to what we have for the other modules